lib/api/debug.go

```diff
@@ -24,5 +21,7 @@ func shouldDebugHTTP() bool {
 func init() {
 	// The debug facility was originally named "http", changed in:
 	// https://github.com/syncthing/syncthing/pull/5548
-	l.SetDebug("api", strings.Contains(os.Getenv("STTRACE"), "api") || strings.Contains(os.Getenv("STTRACE"), "http") || os.Getenv("STTRACE") == "all")
+	if l.IsTraced("http") {
```

Comment:
I think we can remove this these days.

Reply:
I would keep it: The change isn't that old and it doesn't really hurt.

Reply:
But removing it does not cause damage